### PR TITLE
feat: install OpenJDK instead of Oracle JDK

### PR DESCRIPTION
**Pull request by rosen-vladimirov**
From January 1st, 2019, new Oracle's JDK versions will be paid. Migrate the installation of JDK to OpenJDK instead of Oracle JDK.
Move the installation of OpenJDK after installation of Android SDK, as currently the installation of SDK 26.1.1 installs Oracle JDK 1.8 as well.

### CODE REVIEW

File: native-script.ps1 [hunk 1]
```diff
@@ -107,12 +105,26 @@ if (!$env:ANDROID_HOME) {
...
-	$env:JAVA_HOME = $javaHome;
-	refreshenv
+	$openJdkLocation = "C:\Program Files\OpenJDK";
+	$openJdkLocationExists = Test-Path $openJdkLocation;
+
+	if ($openJdkLocationExists -eq $True) {
+		# We should never come here, the installation of OpenJDK should set the variable
+		Write-Host -ForegroundColor DarkYellow "Setting up JAVA_HOME to $openJdkLocation";
+		[Environment]::SetEnvironmentVariable("JAVA_HOME", $openJdkLocation, "User");
+		$env:JAVA_HOME = $javaHome;
```
Comment (Fatme) [inline native-script.ps1]: Shouldn't be
```
$env:JAVA_HOME = $openJdkLocation
```

File: native-script.ps1 [hunk 2]
```diff
@@ -107,12 +105,26 @@ if (!$env:ANDROID_HOME) {
...
+Import-Module "$env:ChocolateyInstall\helpers\chocolateyProfile.psm1"
+refreshenv
+
 if (!$env:JAVA_HOME) {
-	$curVer = (Get-ItemProperty "HKLM:\SOFTWARE\JavaSoft\Java Development Kit").CurrentVersion
-	$javaHome = (Get-ItemProperty "HKLM:\Software\JavaSoft\Java Development Kit\$curVer").JavaHome
-	[Environment]::SetEnvironmentVariable("JAVA_HOME", $javaHome, "User")
-	$env:JAVA_HOME = $javaHome;
-	refreshenv
+	$openJdkLocation = "C:\Program Files\OpenJDK";
```
Comment (Fatme) [inline native-script.ps1]: Is this location fine for both x86 and 64 bit os?

Comment (rosen-vladimirov) [reply native-script.ps1]: No idea :smile:  I'll check it, thanks

Comment (rosen-vladimirov) [reply native-script.ps1]: I've checked it - the path is correct.

**Comment by dtopuzov** [comment]
Closing as it is part of https://github.com/NativeScript/setup-scripts/pull/25

**Pull request closed without merging**